framework/Optimizers/RavenSampled.py

```diff
@@ -419,6 +443,7 @@ def _resolveNewOptPoint(self, traj, rlz, optVal, info):
     if acceptable in ['accepted', 'first']:
       # record history
       self._optPointHistory[traj].append((rlz, info))
+      self.incrementIteration(traj)
```

Comment:
Is this addition true for all RavenSampled? It seems odd that this didn't come up during the opt rework branch testing. I would check to make sure there aren't skipped iterations in the optimizer solution exports for the standard tests due to this line.

Reply:
I will double-check, but I guess I did that when I was discussing it with Andrea

Reply:
I have removed it